com.obsproject.Studio.Plugin.SourceClone.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.SourceClone</id>
+  <extends>com.obsproject.Studio</extends>
```

Comment:
can you add developer name?

Reply:
in that case it would be different from my other plugins like https://github.com/flathub/com.obsproject.Studio.Plugin.MoveTransition/blob/master/com.obsproject.Studio.Plugin.MoveTransition.metainfo.xml

Reply:
that's fine, you can add a developer name to those too.